### ISSUE TITLE
VQA finetune

### ISSUE THREAD

**Issue by 1144181135** [microsoft/Oscar#90]
Hi Oscar teams:
I read your code and find that some parser_argument may loss, like "--model_name_or_path vinvl/model_ckpts/vqa/base/checkpoint-2000000" and "--tokenizer_name". Can you provide this file?
Thanks.

**Comment by CCYChongyanChen** [comment]
The "checkpoint-2000000" can be downloaded at this link: "https://github.com/microsoft/Oscar/blob/master/VinVL_DOWNLOAD.md".
![image](https://user-images.githubusercontent.com/21120604/125780894-0c257d35-bd68-4af2-ac10-dd94ed9bc0fe.png)

Have no idea with "tokenizer_name".The demo code does not have that arg. Maybe you could try without having this arg.

`python oscar/run_vqa.py -j 4 --img_feature_dim 2054 --max_img_seq_length
    50 --data_label_type mask --img_feature_type faster_r-cnn --data_dir vinvl/datasets/vqa
    --model_type bert --model_name_or_path vinvl/model_ckpts/vqa/base/checkpoint-2000000
    --task_name vqa_text --do_train --do_lower_case --max_seq_length 128 --per_gpu_eval_batch_size
    256 --per_gpu_train_batch_size 32 --learning_rate 5e-05 --num_train_epochs 25
    --output_dir results --label_file datasets/vqa/cache/trainval_ans2label.pkl
    --save_epoch 1 --seed 88 --evaluate_during_training --logging_steps 4000 --drop_out
    0.3 --weight_decay 0.05 --warmup_steps 0 --loss_type bce --img_feat_format pt 
    --classifier linear --cls_hidden_scale 3 --txt_data_dir vinvl/datasets/vqa
`

**Comment by jiuge-sun** [comment]
> vinvl/model_ckpts/vqa/base/checkpoin
我现在bert-base，但是不知道怎样使用，请问可以说清楚点吗
